db/migrate/20200714191914_add_nasa_email_field_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddNasaEmailFieldToUsers < ActiveRecord::Migration
+  disable_ddl_transaction!
+
+  def change
+    add_column :users, :nasa_email_communication, :boolean
+    add_index :users, :nasa_email_communication, where: "nasa_email_communication = true", algorithm: :concurrently
```

Comment:
It's the same as the `beta_email_communication` index. Are they supposed to be used differently?

Reply:
i don't know - there is no code in here that actually queries the data with AR. Just make sure the index matches for the query you will write (test the sql in a local test db). If it doesn't it will use up DB RAM and not help the query.

Reply:
Again, I'm making the assumption that this field will be used the same way as the other email list fields, which have the same kind of indexes: 

https://github.com/zooniverse/panoptes/blob/master/db/migrate/20150605103339_add_beta_email_to_users.rb#L5-L6

If the mailer service is filtering users on these fields then it makes sense to use that predicate, and the pressure to get this finished indicates to me that it'll be used pretty soon.

Reply:
I'm not totally familiar with the email list update + synchronization tools (which are not required immediately, as I'll manage the syncing manually to begin; I'll issue up the addition of list syncing to the mailing list worker for the future), but Zach's assumption to mimic the beta email list is the same one Mark and I have adopted when considering the PFE side of things.

And yes, this feature will get immediate use: planning to start highlighting the list and pointing people to the opt-in by the end of next week.  The PFE side of the feature is complete: see https://github.com/zooniverse/Panoptes-Front-End/pull/5747.